sktime/forecasting/naive.py

```diff
@@ -20,26 +20,36 @@ class NaiveForecaster(OptionalForecastingHorizonMixin, BaseLastWindowForecaster)
 
     Parameters
     ----------
-    strategy : str{"last", "mean", "seasonal_last"}, optional (default="last")
+    seasonal : True or False or None, optional (default=None)
```

Comment:
I think `seasonal` should be True or False (not None), default should be False. Or what did you have in mind with None?

Reply:
Will change it.